appmanager.ipynb

```diff
@@ -76,12 +76,17 @@
     "        if app_select.value:\n",
     "            display_app_box(app_select.value)\n",
     "\n",
-    "app_select = ipw.Select()\n",
-    "update_app_list()\n",
+    "app_select = ipw.Select(layout={'height': '{}px'.format(len(registry)*20)})\n",
     "app_select.observe(on_app_change, names='value')\n",
```

Comment:
Would `overflow_y: 'visible'` work instead of computing the height?
http://ipywidgets.readthedocs.io/en/latest/examples/Widget%20Styling.html#Flexbox

Reply:
The options don't overflow the container so setting `overflow_y: visible` has no effect  and the `size` attribute is not exposed by the widget so we can't adjust it.

https://stackoverflow.com/questions/9943886/make-multiple-select-to-adjust-its-height-to-fit-options-without-scroll-bar